sched/timer/timer.h

```diff
@@ -50,6 +50,7 @@ struct posix_timer_s
 {
   FAR struct posix_timer_s *flink;
 
+  clockid_t        pt_clock;       /* Specifies the clock to use as the timing base. */
```

Comment:
Would this be clearer?
```suggestion
  clockid_t        pt_clockid;       /* Specifies the clock to use as the timing base. */
```

Reply:
Hi @davids5 ,

Thanks for review! The naming of pt_clock actually follows the style of Linux kernel, 

https://github.com/torvalds/linux/blob/f8394f232b1eab649ce2df5c5f15b0e528c92091/include/linux/posix-timers.h#L215

So, I prefer the kernel style :)